deploy/craned/deployment.yaml

```diff
@@ -215,7 +215,6 @@ data:
         add_header Cache-Control "must-revalidate";
         add_header Cache-Control "max-age=300";
         add_header ETag "1.90.0-rc.0";
-        add_header Access-Control-Allow-Origin *;
```

Comment:
/grafana required this header.
please add 'add_header Access-Control-Allow-Origin *;' in line 230 @lbbniu